# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515182819 28800
#      Fri Jan 05 12:06:59 2018 -0800
# Node ID e1295dd49c71b89d2dbaed675b7ba5fafc7dd99e
# Parent  abd196490ed31f84e80401628469e930da7200ac
build: Don't delete module.mk for doxyhtml-maintainer-clean rule.

* doc/doxyhtml/module.mk: Use '$$' to get a single '$' in shell command.
Add 'module.mk' to list of patterns for files that are not removed.

diff --git a/doc/doxyhtml/module.mk b/doc/doxyhtml/module.mk
--- a/doc/doxyhtml/module.mk
+++ b/doc/doxyhtml/module.mk
@@ -8,16 +8,16 @@ DOXYGEN_PAGES = \
   %reldir%/pages/macros.dox \
   %reldir%/pages/README.md
 
 doxyhtml: %reldir%/Doxyfile $(DOXYGEN_PAGES) | %reldir%/$(octave_dirstamp)
 	doxygen %reldir%/Doxyfile
 
 doxyhtml-maintainer-clean:
 	rm -f doc/doxygen_sqlite3.db
-	rm -rf `ls -d %reldir%/* 2>/dev/null | $(GREP) -v 'Doxyfile\.in\|README\|pages$'`
+	rm -rf `ls -d %reldir%/* 2>/dev/null | $(GREP) -v 'module\.mk\|Doxyfile\.in\|README\|pages$$'`
 
 doc_EXTRA_DIST += \
   $(DOXYGEN_PAGES) \
   %reldir%/Doxyfile.in \
   %reldir%/README
 
 DIRSTAMP_FILES += %reldir%/$(octave_dirstamp)
